deployments/fluxcd/values.yaml

```diff
@@ -9,5 +9,6 @@ registry:
 git:
   url: git@github.com:hmcts/cnp-flux-config
   pollInterval: 1m
+  secretName: flux-git-deploy
```

Comment:
how does this stop us needing to restart it? the problem before was after a change it didn't reload it

Reply:
Currently, we install flux, create secret , then restart flux. Even on a clean install, we are restarting it after creating secret. I have moved secret creation upfront so that we don't need to restart. Also, with current setup, reinstalling just flux through pipeline will fail as secret already exists 

https://dev.azure.com/hmcts/CNP/_releaseProgress?_a=release-environment-logs&releaseId=1835&environmentId=6669